### ISSUE TITLE
Fork started to crash on one specific repository

### ISSUE THREAD

**Issue by imsh** [ForkIssues/TrackerWin#1194]
As soon as I open one specific repository, fork crashes. I was using fork with this repository before and everything was working fine and I kept it in my workspace, but I don't look at this repo often, so I'm not sure if the latest fork version ever worked with it.

Fork version is 1.62.1.0
Git version used: Crashes both with embedded version and with 2.32.0 version of git

Here is the [fork.log](https://github.com/fork-dev/TrackerWin/files/6630606/fork.log)

Exception from the Application log:
```
Application: Fork.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NullReferenceException
   at Fork.UI.Controls.HighlightingTextBlockExtensions+<>c__DisplayClass0_0.<ApplySearchAndButrackerHighlighting>b__1(Int32, Int32, Fork.CountableRange, Fork.CountableRange)
   at Fork.CountableRangeMerger.Merge(Fork.CountableRange[], Fork.CountableRange[], System.Action`4<Int32,Int32,Fork.CountableRange,Fork.CountableRange>)
   at Fork.UI.Controls.HighlightingTextBlockExtensions.ApplySearchAndButrackerHighlighting(System.Windows.Controls.TextBlock, System.String, Fork.BugtrackerLinkDefinition[])
   at Fork.UI.UserControls.RevisionDetailsBarUserControl.SetRevisionDetails(Fork.Git.RevisionDetails, Fork.UI.Controls.AvatarImage, System.Windows.Controls.TextBlock, System.Windows.Controls.TextBlock, System.Windows.Controls.TextBlock, System.Windows.Controls.TextBlock, Fork.BugtrackerLinkDefinition[])
   at Fork.UI.UserControls.RevisionDetailsBarUserControl.SetRevisionDetails(Fork.Git.RevisionDetails, Fork.BugtrackerLinkDefinition[], Fork.Git.RevisionDetails)
   at Fork.UI.UserControls.RevisionDetailsUserControl+<>c__DisplayClass19_1.<UpdateRevisionDetails>b__1()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(System.Object)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   at System.Windows.Threading.Dispatcher.PushFrame(System.Windows.Threading.DispatcherFrame)
   at System.Windows.Application.RunDispatcher(System.Object)
   at System.Windows.Application.RunInternal(System.Windows.Window)
   at System.Windows.Application.Run(System.Windows.Window)
   at Fork.App.Main()

Exception Info: System.Reflection.TargetInvocationException
   at System.RuntimeMethodHandle.InvokeMethod(System.Object, System.Object[], System.Signature, Boolean)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(System.Object, System.Object[], System.Object[])
   at System.Reflection.RuntimeMethodInfo.Invoke(System.Object, System.Reflection.BindingFlags, System.Reflection.Binder, System.Object[], System.Globalization.CultureInfo)
   at <Module>.Main(System.String[])
```

**Comment by DanPristupov** [comment]
There is a problem with a bugtracker rule.
The local issue tracker rule is stored in `.git/issuetracker` file. Can you share what regular expression you use? Then you can probably remove it.

**Comment by imsh** [comment]
Here is the file: [issuetracker.txt](https://github.com/fork-dev/TrackerWin/files/6630740/issuetracker.txt)
I've added .txt so that it can be uploaded to github and renamed repo names there.

I see that the it has the regex in the url section, this is quite strange :) I don't know if I added it there myself in ui. I'm quite sure I was not editing the file directly. When I removed the "(\\d+)" from the url section, the fork has stopped crashing 👍

The problem is solved for me.
